src/sentry/interfaces/stacktrace.py

```diff
@@ -33,6 +33,12 @@
     [a-f0-9]{40}       # sha1
 )/""", re.X | re.I)
 
+# Java Spring specific anonymous classes.
+# see: http://mydailyjava.blogspot.co.at/2013/11/cglib-missing-manual.html
+_java_enhancer_re = re.compile(r'''
+(\$\$[\w_]+?CGLIB\$\$)[a-fA-F0-9]+
```

Comment:
This can also drop in a `_[0-9]` after the hash code: https://github.com/cglib/cglib/blob/master/cglib/src/main/java/net/sf/cglib/core/DefaultNamingPolicy.java#L46-L47